README.md

```diff
@@ -1,19 +1,61 @@
 # Open Trading Research Infrastructure
-Open Trading Research Infrastructure.
 
-## config.json
+**Open Trading Research Infrastructure (OTRI)** is a research platform for financial computing focusing on data retrieval, validation, and analysis.
+
+## Getting started
+
+### Requirements
+
+- Any OS that supports Python 3.7
+- Python 3.7 and PIP
+
+### Installation
+
+```bash
```

Comment:
Codacy found an issue: [[no-shell-dollars] Do not use dollar signs before shell commands](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5985894)